### ISSUE TITLE
test_request_loadButtonData() failing as likely result of #1943

### ISSUE THREAD

**Issue by cgolubi1** [buttonmen-dev/buttonmen#1999]
Okay, i am super confused.  When i run phpunit tests in virtualbox, i'm now getting failures like:
```
There was 1 failure:

1) responderTest::test_request_loadButtonData
Failed asserting that 706 matches expected 711.

/buttonmen/test/src/api/responderTest.php:1604
```

As well i should be!  #1943 added 5 additional buttons, for a total of 711, but `test_request_loadButtonData()` is still asking for 706 buttons.

So the test should be updated.  But how did #1943 pass jenkins (and it did)?

**Comment by cgolubi1** [comment]
Yeah, so as you can see, a build i just tried to run - http://jenkins.buttonweavers.com:8080/job/buttonmen-cgolubi1/440/ - did indeed fail exactly as i would have expected.  So i'm going to fold a fix into that pull, but i really don't understand why shadowshade's builds for #1943 didn't detect the problem.

**Comment by blackshadowshade** [comment]
So, I know why this happened.

Before your fix in #2000 was approved by me, the last change to that line of code was by you, with git blame giving the following info:

89a63888 (Chaos Golubitsky 2016-02-15 23:31:16 -0500  1604)

Thus, the newly-failing unit test was introduced by you when you migrated the unit tests entirely to responderTest in #1952.

My last merge with master on this pull request was on Feb 6, commit 259eea0, and I note that at this commit, this particular test case counting the number of buttons didn't seem to exist at all.

Thus, the fail comes because I didn't merge master after you did that big migration of unit tests. Looks like I'll be doing a mass merge of all remaining pull requests to bring them back up to date.

**Comment by cgolubi1** [comment]
Ahh, okay, thanks for the explanation, that makes sense.
